### ISSUE TITLE
Feature Request：Create a desktop icon when installing an app 

### ISSUE THREAD

**Issue by pingmoogle** [ScoopInstaller/Scoop#4212]
For example, you can install an app and create a desktop shortcut at the same time with the following command.
```
scoop install someapp --desktopicon
```
or
```
scoop install someapp -d
```

Thanks for considering.

**Comment by HiKE-yt** [comment]
Yes ! It MUST be considered, it's a very practical feature, accessing things on the desktop is so much better since Windows 10/11 startup panel.

**Comment by kunago** [comment]
In the meantime, it's possible to make use of manifest properties "post_install" and "pre_uninstall". I am currently using those 2 properties to create and remove desktop icons and it works perfect.

**Comment by xrgzs** [comment]
What about `scoop reset`, `scoop uninstall`, `scoop update` ? Is there any idea?

How to manage desktop icons manually created by `scoop install -d` ?

**Comment by xrgzs** [comment]
I implemented automatic creation of desktop shortcuts in my fork of Scoop by configuring `scoop config desktop_shortcut true`. I reused the method from startmenu shortcuts creation to make the modification minimal. It works well. 

https://github.com/xrgzs/scoop/tree/feat/desktop-shortcut

However, this will create desktop shortcuts every time you do `scoop install/update/reset/uninstall`.  So if an installed app updates, which user do not want to see it in the desktop, it would lead to a bad user experience.

And the second command in #5311 can be implement by `scoop reset firefox`. Or just drag it to desktop from startmenu. I don't think it's necessary to add a set of desktop shortcuts management to Scoop.

**Comment by xrgzs** [comment]
There is a solution to display the icons of installed apps in the desktop which I am using. You can install a desktop organizer and map the `Scoop Apps` folder to desktop. You can close the desktop organizer when you don't want to see it. I think this is more in line with Scoop's philosophy.
